test/cpp/jit/test_misc.cpp

```diff
@@ -70,8 +68,11 @@
 
 namespace torch {
 namespace jit {
-c10::OperatorOptions aliasAnalysisFromSchema();
-namespace test {
+c10::OperatorOptions aliasAnalysisFromSchema() {
```

Comment:
Where was it defined previously? I couldn't find it in the diff.

Reply:
Oh I just mechanically was changing things, I’ll clean it up in a follow up if it’s unused

Reply:
I was just curious because this patch added a new definition, but didn't remove any existing.